### DISCUSSION TITLE
My GitHub pages portfolio still shows up even though I deleted the repository

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47891] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey friends, I created a portfolio site a few years ago using GitHub Pages. About 6 months ago, I unpublished it and deleted the repository. Yet, my portfolio still shows up in search results when I Google it. I’m at a loss. How do I actually delete it? 

![6D0D46BB-BCA5-449C-A0CA-26B199EFC0F3](https://user-images.githubusercontent.com/48967489/219880124-1691db7f-edf3-46e3-8cab-21a529e24b2b.png)

**Comment by melvincwng** [comment, marked as answer]
@busyhero, this documentation link from Google might be useful for your issue:

https://support.google.com/websearch/answer/11080680?hl=en

Hope this helps 😃!

**Reply by ghost** [reply]
Thank you! I'll try that. :)

**Reply by surakshitha** [reply]
Did it work @busyhero ??

**Reply by ghost** [reply]
Unfortunately not. :(
I tried the solution that I think the article was recommending, but it was a bit confusing for a non-technical person like me, and I’m not sure I followed all the instructions correctly. 

I can still access my portfolio by typing in the url, and it shows up in my search results.

**Comment by perabjoth** [comment]
Create a github respository with the same exact name as the original repository. Add any random index.html file to the repo and deploy github pages using root directory. Then once it publishes, you can unpublish as you normally do [https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site](url) and then delete the repository.
